dev-docs/guides/wires.md

```diff
@@ -13,11 +13,11 @@ Wires also have an associated `CanonicalPendingTransaction`, the `amount_cents`
 
 The most up to date amount for a wire (in USD) is accessible using `Wire#usd_amount_cents`. `Wire#amount_cents` and `Wire#currency` store the currency and amount of the wire, in that currency.
 
-## How we collect specific country fields
+## How we collect country-specific fields
 
 Column lists out the specific details needed for a wire in each country: [column.com/docs/international-wires/country-specific-details](https://column.com/docs/international-wires/country-specific-details).
 
-We’ve translated this into the `Wire#information_required_for` method. Which you pass in a country code and are returned an array of details that need to be collected. This information is stored in the `recipient_information` JSONB column of the database, we use [`ActiveRecord::Store`](https://api.rubyonrails.org/classes/ActiveRecord/Store.html) for this. On the frontend, we conditionally render these fields using Alpine based on the recipient’s country. 
+We’ve translated this into the `Wire#information_required_for` method in which you pass a country code and are returned an array of details that need to be collected. This information is stored in the `recipient_information` JSONB column of the database. We use [`ActiveRecord::Store`](https://api.rubyonrails.org/classes/ActiveRecord/Store.html) for this. On the frontend, we conditionally render these fields using Alpine based on the recipient’s country. 
```

Comment:
```suggestion
We’ve translated this into the `Wire#information_required_for` method; you can pass it a country code and it will return an array of details that need to be collected. This information is stored in the `recipient_information` JSONB column of the database. We use [`ActiveRecord::Store`](https://api.rubyonrails.org/classes/ActiveRecord/Store.html) for this. On the frontend, we conditionally render these fields using Alpine based on the recipient’s country. 
```